apps/src/aichat/aichatApi.ts

```diff
@@ -10,7 +10,7 @@ import {ValueOf} from '../types/utils';
 import {chatHistoryValidator} from './api/validators';
 import {
   AiCustomizations,
-  AichatContext,
+  ApiContext,
```

Comment:
`Api` feels a bit generic?

Reply:
Yeah that's fair, maybe it's fine to just leave this as `aichatContext` as its still only used in `aichat` code after all?